CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.7.0 (2020-09-29)
+  - Add stubborn network call helpers.
+  - Update default version to v7.0.
+  - Switch from TravisCI to CircleCI.
+  - Remove coveralls.
+  - Require ruby 2.5+.
```

Comment:
Note that Ruby 2.4 and lower have hit end-of-life already, so 2.5 is the lowest supported 2.x version.